packages/client/src/v2-events/features/events/actions/quick-actions/register.ts

```diff
@@ -21,7 +21,10 @@ export const register: QuickActionConfig = {
     }
   },
   onConfirm: ({ event, actions, customActions }) => {
-    if (event.status === EventStatus.enum.DECLARED) {
+    // @TODO: This is a temporary measure, until the VALIDATE-action is turned in to a custom action.
+    // At that point we should remove all logic and mentions of the validate action in core.
+    // Related issue: https://github.com/opencrvs/opencrvs-core/issues/11225
```

Comment:
This is also the reason why 'Validated' was not made into a InherentFlag or any constant in core